Makefile

```diff
@@ -185,107 +184,227 @@ test-integration: test-integration-aws test-integration-clickhouse test-integrat
+	$(CONTAINER_TOOL) run -d --network=test-integration-aws -p 4568:4568 -p 4572:4572 -p 4582:4582 -p 4571:4571 -p 4573:4573 --name localstack -e SERVICES=kinesis:4568,s3:4572,cloudwatch:4582,elasticsearch:4571,firehose:4573 localstack/localstack@sha256:f21f1fc770ee4bfd5012afdc902154c56b7fb18c14cf672de151b65569c8251e
+	$(CONTAINER_TOOL) run -d --network=test-integration-aws -p 6000:6000 --name mockwatchlogs -e RUST_LOG=trace luciofranco/mockwatchlogs:latest
+	sleep 60 # Many services are very slow... Give them a sec...
+
+stop-integration-aws:
+	$(CONTAINER_TOOL) rm --force ec2_metadata mockwatchlogs localstack 2>/dev/null; true
+	$(CONTAINER_TOOL) network rm test-integration-aws 2>/dev/null; true
+
 test-integration-aws: ## Runs AWS integration tests
 ifeq ($(AUTOSPAWN), true)
-	${MAYBE_ENVIRONMENT_EXEC} $(CONTAINER_TOOL)-compose up -d dependencies-aws
-	sleep 5 # Many services are very lazy... Give them a sec...
+	$(MAKE) -k stop-integration-aws \
+    ; rc=$$? \
+	$(MAKE) start-integration-aws
 endif
 	${MAYBE_ENVIRONMENT_EXEC} cargo test --no-default-features --features aws-integration-tests --lib ::aws_ -- --nocapture
 ifeq ($(AUTODESPAWN), true)
-	${MAYBE_ENVIRONMENT_EXEC} $(CONTAINER_TOOL)-compose stop
+	$(MAKE) -k stop-integration-aws
 endif
 
+start-integration-clickhouse:
+	$(CONTAINER_TOOL) network create test-integration-clickhouse
+	$(CONTAINER_TOOL) run -d --network=test-integration-clickhouse -p 8123:8123 --name clickhouse yandex/clickhouse-server:19
+
+stop-integration-clickhouse:
+	$(CONTAINER_TOOL) rm --force clickhouse 2>/dev/null; true
+	$(CONTAINER_TOOL) network rm test-integration-clickhouse 2>/dev/null; true
+
 test-integration-clickhouse: ## Runs Clickhouse integration tests
 ifeq ($(AUTOSPAWN), true)
-	${MAYBE_ENVIRONMENT_EXEC} $(CONTAINER_TOOL)-compose up -d dependencies-clickhouse
-	sleep 5 # Many services are very lazy... Give them a sec...
+	$(MAKE) -k stop-integration-clickhouse \
+    ; rc=$$? \
+	$(MAKE) start-integration-clickhouse
+	sleep 30 # Many services are very slow... Give them a sec...
 endif
 	${MAYBE_ENVIRONMENT_EXEC} cargo test --no-default-features --features clickhouse-integration-tests --lib ::clickhouse:: -- --nocapture
 ifeq ($(AUTODESPAWN), true)
-	${MAYBE_ENVIRONMENT_EXEC} $(CONTAINER_TOOL)-compose stop
+	$(MAKE) -k stop-integration-clickhouse
 endif
 
 test-integration-docker: ## Runs Docker integration tests
 	${MAYBE_ENVIRONMENT_EXEC} cargo test --no-default-features --features docker-integration-tests --lib ::docker:: -- --nocapture
 
+start-integration-elasticsearch:
+	$(CONTAINER_TOOL) network create test-integration-elasticsearch
+	$(CONTAINER_TOOL) run -d --network=test-integration-elasticsearch -p 4568:4568 -p 4572:4572 -p 4582:4582 -p 4571:4571 -p 4573:4573 --name localstack -e SERVICES=kinesis:4568,s3:4572,cloudwatch:4582,elasticsearch:4571,firehose:4573 localstack/localstack@sha256:f21f1fc770ee4bfd5012afdc902154c56b7fb18c14cf672de151b65569c8251e
```

Comment:
Do you think the readability would be improved by breaking these into multiple lines?